src/components/Containers/ContainerCard.tsx

```diff
@@ -10,6 +10,7 @@ interface ContainerCardProps {
 
 const ContainerCard = ({ container, onClick, isSelected }: ContainerCardProps) => {
     const statusColor = !container.isRunning() ? 'bg-red-500' : 'bg-green-500';
+    const containerImage = container.Image.split(':')[0];
```

Comment:
Instead of doing it here you can add new method in the `Container` model to get the image name.

Reply:
Good idea. Fixed, thank you